public/javascripts/payments.js

```diff
@@ -131,6 +132,33 @@
   // Mount the iDEAL Bank Element on the page.
   idealBank.mount('#ideal-bank-element');
 
+  /**
+   * Add a BECS element that matches the look-and-feel of the app.
+   * 
+   * This allows you to collect australian bank account details
+   */
+
+  const becsBank = elements.create('auBankAccount', {style})
+
+  // Mount the BECS element on the page.
+  becsBank.mount('#becs-bank-element')
```

Comment:
Missing some semicolons here and on L141.

Reply:
Thanks! I've added these.